src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates.  All rights reserved.
```

Comment:
You do have the copyright header here, good! :-) But it's lacking the year, like so:
 * Copyright (c) 2021, Amazon.com, Inc. or its affiliates. All rights reserved.